lib/roger_jshint/generator.rb

```diff
@@ -0,0 +1,44 @@
+require 'open-uri'
+
+require 'roger/cli/generate'
+require 'roger/cli/command'
+require 'roger/generators'
+
+module RogerJsHint
+  # Lint configuration retriever
+  class Generator < Roger::Generators::Base
+    include Thor::Actions
+    CONFIG_PATH = '.jshintrc'
+    DEFAULT_CONFIG_URL = 'https://raw.githubusercontent.com/DigitPaint/javascript/master/linters/jshintrc'
+
+    desc 'JsHint generator creates an .jshintrc config file'
+    class_option(
+      :config,
+      type: :string,
+      aliases: ['-c'],
+      desc: "Optional config file to use takes a path or url, by
+             default it uses the company' default"
+    )
+
+    class_option(
+      :force,
+      type: :boolean,
+      aliases: ['-f'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.